source/main/cardano/CardanoWalletLauncher.js

```diff
@@ -66,18 +78,66 @@ export async function CardanoWalletLauncher(
       break;
   }
 
-  Logger.info('Starting Node now...', { path, walletArgs });
-  const childProcess = spawn(path, walletArgs, {
-    stdio: walletStdio,
-    env: {
-      // $FlowFixMe
-      ...process.env,
-      ...envVariables,
+  Logger.info('Starting Node now...', { path, walletArgs, walletOpts });
+
+  const launcher = new cardanoLauncher.Launcher({
```

Comment:
Looks correct for jormungandr.

Although this sets up the launcher it doesn't actually start anything until `start()` is called. The previous code appears to spawn the wallet process.

Reply:
👍